maro/simulator/scenarios/finance/README.md

```diff
@@ -0,0 +1,150 @@
+# Finance scenario in MARO
+
+## Introduction
+
+The Finance scenario simulates securities trading in financial markets.
+Imagine you invest in multiple finance markets. For example, US NASDAQ and CN Shanghai:
+You can create a strategy which decides how to buy and sell stocks in these twoindividual markets, and when NASDAQ about to go down, transfer your investments to Shanghai.
+So, your investment can keep increasing no matter how the market goes up or down.
+
+## Environment Modeling
+
+To simulate real markets, we implement the typical order types, cost of the trading, leverage
+rate of account, and dynamic exchange rates between different currencies.
+
+A typical trading process is:
+
+![Order_Workflow](../../../../docs/source/images/order.png "Order_Workflow")
+
+A typical transfer process is:
+
+![Transfer_Workflow](../../../../docs/source/images/transfer.png "Transfer_Workflow")
```

Comment:
If transfer not available in this new version, remove all description related to transfer

Reply:
done